core/mcp.go

```diff
@@ -194,6 +194,91 @@ func (m *MCP) tools(srv *dagql.Server, typeName string) ([]LLMTool, error) {
 			// never a reason to call "sync" since we call it automatically
 			continue
 		}
+		// Hide functions from the largest and most commonly used core types,
```

Comment:
... so this is a allowlist? hard to tell from the diff what exactly is being hidden.

Reply:
No this is a blocklist. The explicit list is of functions to hide

Reply:
@shykes would you be opposed (either here or a follow-up) to having this as part of the schema declaration? So we'd do `dagql.Func(...).Doc(...).ExposeToLLM()` or something (name doesn't really matter to me).

Otherwise, it feels like it's easy for the schema to drift from this list over time, which would be nice to avoid.